index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Wuyep Abdul</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="nav-bar align-center d-flex space-between">
+      <a class="padding-round">John Doe</a>
+      <nav class="bars"><img src="./images/bars.png" alt="image" /></nav>
+    </header>
+
+    <main>
+      <section class="headline d-flex align-center">
+        <div class="headline-ellipse">
+          <header>
+            <h1>Laura Jong</h1>
+            <h2>Software Developer</h2>
+          </header>
```

Comment:
- [ ] It would be better if you use the `div` tag instead of the `header` because you can use the `header` tag in your header section and you used it. But now in there, you are using the `header` tag inside the `main` tag. It can cause losing Semantic HTML. To see more about semantic HTML, click this [link](https://www.freecodecamp.org/news/semantic-html5-elements/) 👏.